File: Giovanna-e-Gabriela/CPP/Lista-1/ex-03.cpp
/*
Data: 05/05/2026
Autores: Gabriela Onaga e Giovanna Costa
Descrição:Ler vários números até que seja digitado um número negativo. Calcular e exibir a soma desses números.
*/

#include <iostream>
using namespace std;

int main() {
    double numero, soma = 0;

    cout << "Digite vários numeros:"<< endl;

    while (true) {
        cin >> numero;

        if (numero < 0) {
            break;
        }

        soma += numero;
    }

    cout << "Soma total: " << soma << endl;

    return 0;
}

File: Giovanna-e-Gabriela/Python/python-conversion/python/ex03.py
"""
    Autor: Giovanna Cabral e Gabriela Onaga
    Data: 11/08/2026
    Descrição: Lê um número inteiro positivo e exibe a contagem de 1 até esse número.
"""

num = int(input("Digite um número inteiro positivo: "))
contador = 1

if num > 0:
    while contador <= num:
        print(contador)
        contador += 1

else: 
    print("Numero invalido. Digite um valor positivo.") 
